packages/starlight/translations/zh-CN.json

```
@@ -18,6 +18,7 @@
   "page.lastUpdated": "最近更新：",
   "page.previousLink": "上一页",
   "page.nextLink": "下一页",
+  "page.draft": "This content is a draft and will not be included in production builds.",
```

```suggestion
  "page.draft": "此内容为草稿，不会包含在生产版本中。",
```
Nice work! I guess it would be great to have Chinese translations when merging

---

Thanks for adding the translation ahead of time, super appreciated :raised_hands: (note that I may ping you if the initial text gets modified before the release)